eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1992,4 +1992,10 @@
       <Class name="~com\.azure\.search\.documents\.indexes\.models\.(EdgeNGramTokenFilter|KeywordTokenizer|LuceneStandardTokenizer|NGramTokenFilter)"/>
       <Bug pattern="URF_UNREAD_FIELD"/>
   </Match>
+
+  <!-- Use static final ObjectMapper property to save the cost of ObjectMapper initialization. -->
+  <Match>
+    <Class name="com.azure.search.documents.SearchAsyncClient" />
```

Comment:
Don't think this is required any longer as `SearchAsyncClient` no longer has an inner anonymous class, right?

Reply:
Spotbugs complaints about the inner static fields operation as well.